perception/compare_map_segmentation/include/compare_map_segmentation/compare_elevation_map_filter_node.hpp

```diff
@@ -53,7 +53,7 @@ class CompareElevationMapFilterComponent : public pointcloud_preprocessor::Filte
   void elevationMapCallback(const grid_map_msgs::msg::GridMap::ConstSharedPtr elevation_map);
 
 public:
-  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+  PCL_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
Could you write some reference links?

Reply:
https://github.com/PointCloudLibrary/pcl/blob/3f3038a473a9d97a117822aafe6841b0a9c8612e/common/include/pcl/memory.h#L54-L65